docs/harfbuzz-sections.txt

```diff
@@ -482,6 +482,8 @@ hb_ot_layout_collect_features
 hb_ot_layout_feature_get_lookups
 hb_ot_layout_feature_with_variations_get_lookups
 hb_ot_layout_get_attach_points
+hb_ot_layout_get_feature_characters
```

Comment:
Maybe hb_ot_layout_feature_get_characters and hb_ot_layout_feature_get_name_ids now that we are passing feature index also?

Reply:
I think, yes.